firefox_puppeteer/tests/test_toolbars.py

```diff
@@ -0,0 +1,132 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette import Wait
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+from firefox_ui_harness.decorators import skip_under_xvfb
+
+
+class TestLocationBar(FirefoxTestCase):
+
+    def test_reload(self):
+        event_types = ["shortcut", "shortcut2", "button"]
+        for event in event_types:
+            # TODO: Until we have waitForPageLoad, this only tests API
```

Comment:
I assume there is no such thing in Marionette what we could use? I was hoping that we get it for free. :/